.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI workflow
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
can you please add this matrix: https://github.com/fastify/fastify/blob/master/.github/workflows/ci.yml#L21

ESM modules behave differently on Unix and windows systems: winodws paths are not valid URIs (since c: is considered a protocol :/ ), while unix ones are. We have  working exports/imports on macOS/Linux but broken on Windows.

Reply:
Added, but I skipped any node less than 14 as esm won't work on those anyway and the rest won't really differ